app/Models/Beatmapset.php

```diff
@@ -742,7 +735,7 @@ public function recentEvents()
 
     public function status()
     {
-        return static::STATES[$this->approved];
+        return array_search($this->approved, self::STATES);
```

Comment:
if `self::STATES` doesn't contain `$this->approved`, this function will return `false`...